include/glow/Backend/BackendUtils.h

```diff
@@ -120,5 +120,41 @@ class RuntimeBundle {
 };
 } // namespace runtime
 
+/// If \p PH is an output placeholder, \returns true.
```

Comment:
Describe the parameter F like you do for other functions. 

`\returns` should be usually on a new line, IIRC.

Update other Doxygen comments accordingly.

Reply:
> Describe the parameter F like you do for other functions.
> 
> `\returns` should be usually on a new line, IIRC.
> 
> Update other Doxygen comments accordingly.

OK. I made the updates and put the `\returns`  into new line.